components/neeva/browser/content_filter_client.h

```diff
@@ -22,6 +22,7 @@ class ContentFilterClient : public base::SupportsUserData::Data {
   static ContentFilterClient* Get(content::WebContents* web_contents);
   static ContentFilterClient* GetOrCreate(content::WebContents* web_contents);
 
+  // Called in tab_impl.cc, see: TabImpl::SetContentFilterCallbackClient()
```

Comment:
The problem is `set_callback` feels like a generic name that results in tons of matches when doing a code search. (Even VS Code has trouble finding it). Do you have a better way of knowing how to find who calls this function?

Maybe if we rename it to `set_content_filter_callback` it would be easier to find via grep or VS Code

Reply:
I wouldn't solve it that way. Hmm, maybe we are just missing a setup like cs.chromium.org that would solve this problem. I believe github has a code indexer / code search tool as well. Maybe we just need to figure out how to enable that for this repo.

Also, I'd first search for the class name to find which classes use this class. Then look for `set_callback`.

Reply:
```
cd weblayer
grep -ri ContentFilterClient
```